charts/stable/photoprism/values.yaml

```
@@ -16,6 +16,12 @@ persistence:
     enabled: true
     mountPath: "/photoprism_temp"
     type: emptyDir
+  originals:
+    enabled: true
+    mountPath: "/originals"
+  import:
+    enabled: true
+    mountPath: "/import"
```

```suggestion
  originals:
    enabled: true
    mountPath: "/photoprism/originals"
  import:
    enabled: true
    mountPath: "/photoprism/import"
```

Maybe keep those as they were?
That way people would just need to remove the additional storage and just use the "built in" gui to point to their data